on-prem/package.json

```diff
@@ -6,7 +6,7 @@
     "start": "ng serve & ng build common --watch",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test",
+    "test": "ng test --browsers ChromeHeadless --watch=false",
```

Comment:
Careful here... this will also modify a developer's test cases if they run `npm run test` instead of `ng test` ... I normally explicitly make the call as was seen above in the build.yaml, but it seems like something might not have been working there since you modified it to this. Maybe we could chat about the problems seen and fix it together?

Reply:
I thought as much. But yes, a chat would nice to good to fix this together.